src/WebJobs.Script/Extensions/FuncExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class FuncExtensions
+    {
+        public static Func<T, Task> Debounce<T>(this Func<T, Task> func, int milliseconds = 300)
```

Comment:
A bit tricky, since it doesn't show up in the diff, but this is called by the variables we have switched to using `Func` from `Action`. Since there was already a debounce call there, they are not showing up here.